config/manager/manager.yaml

```diff
@@ -47,7 +47,7 @@ spec:
           args:
             - "--port=8443"
             - "--logtostderr"
-            - "--emit-deny-events"
+            - "--emit-admission-events=false"
```

Comment:
isn't default false?

Reply:
yes we want to default to `false`

Reply:
we don't need to set it here if its false